src/main/java/oc/wh40k/units/ty/TYDeathleaper.java

```diff
@@ -10,12 +9,12 @@ public TYDeathleaper() {
         name = "Deathleaper(";
         grundkosten = getPts("Deathleaper");
         power = 4;
-        add(ico = new oc.Picture("oc/wh40k/images/TYTodeshetzer.jpg"));
-        complete();
 
         seperator();
```

Comment:
Is the separator here still needed?

Reply:
No, actually not. Wasn't sure if there was a problem with error text. But they still fit.